pom.xml

```
@@ -45,12 +46,12 @@
         <munit.input.directory>src/test/munit</munit.input.directory>
         <munit.output.directory>${basedir}/target/test-mule/munit</munit.output.directory>
         <mavenResourcesVersion>3.3.1</mavenResourcesVersion>
-        <munit.version>2.3.18</munit.version>
-        <mtf.tools.version>1.1.2</mtf.tools.version>
-        <munit.extensions.maven.plugin.version>1.1.2</munit.extensions.maven.plugin.version>
+        <munit.version>3.1.0-rc3</munit.version>
+        <mtf.tools.version>1.2.0-rc2</mtf.tools.version>
```

build is failing because there's no rc2
```suggestion
        <mtf.tools.version>1.2.0-rc1</mtf.tools.version>
```